queue-health/start-vm.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o verbose
+set -o errexit
+
+NAME='queue-health'
+
+echo 'Copy files...'
+gsutil -m cp * gs://kubernetes-test-history/sq/
+
+echo "Check for previous version of ${NAME}..."
+PREVIOUS="$(gcloud compute instances list --filter="name=${NAME}" || true)"
+if [[ -n "${PREVIOUS}" ]]; then
+  echo "Delete previous version of ${NAME}..."
+  gcloud -q compute instances delete "${NAME}"
+fi
+
+echo "Create ${NAME}..."
+gcloud compute instances create \
+  "${NAME}" \
+  --boot-disk-size=50GB \
```

Comment:
Hmm... I think I'd rather just move to GKE for the next version rather than try too hard to optimize the VM